### ISSUE TITLE
Two links on the Love Your Block page have the same name and different hrefs

### ISSUE THREAD

**Issue by ollie-iterators** [CityOfBoston/Iterators#202]
Screengrab/videos (insert file names here)
2020 11 10, Same name different href for two links.mp4
[2020 11 10, Same name different href for two links.mp4.zip](https://github.com/CityOfBoston/Iterators/files/5519323/2020.11.10.Same.name.different.href.for.two.links.mp4.zip)

Link
https://d8-stg.boston.gov/civic-engagement/love-your-block

Node # (if known)
2861

Type of User:  Anonymous

Environment
(Device > OS > Browser)
Computer > MacOS 10.15.7 > Chrome 86

Trusted Tester Test
Links

Brief Description
Two links on the Love Your Block page have the same name and different hrefs

Steps to reproduce
1)	00:00, Load the URL
2)	00:05, First link with same name, different href
3)	00:07, Second link with same name, different href
4)	00:22, First link’s href is shown to be mailto:loveyourblock@boston.gov
5)	00:32, Second link’s href is shown to be: mailto:loveyourblock@boston.gov?subject=Civic%20Engagement%20Week%202018

Desired Result
•	The links with the same name have the same href values

Actual Result
•	Two links with the same name have different hrefs

**Comment by ollie-iterators** [comment]
Other Links with the same Issue

- Node: 3606 (Link: https://d8-stg.boston.gov/civic-engagement/love-your-block). The links that have the same name and different hrefs are all named '617-343-1027'

- Node: 121 (Link: https://d8-stg.boston.gov/departments/transportation). The links that have the same name and different hrefs are all named 'Events'

**Comment by subaha-cob** [comment]
@ollie-iterators the link on node:121 (Link: https://d8-stg.boston.gov/departments/transportation). The links that have the same name and different hrefs are all named 'Events' -- Does not have the same link. One is a regular URL the other is an anchor link. Are those considered the same link?

**Comment by ollie-iterators** [comment]
This appears to be fixed
